examples/edge_detection/main.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+
+import cv2
+import torch
+
+import kornia as K
+from kornia.contrib import EdgeDetector
+
+
+def my_app():
```

Comment:
Shouldn't it go to tutorials?

Reply:
i can port later, but i would like to have also raw python examples, i use quite often as a template to modify on top of that. Also, the one from opencv/webcam how is that supported in notebooks ?

Reply:
Well, I was more thinking about keeping all examples in a single place, rather than converting to notebook actually...

Reply:
ok, we can make later and it all the examples at once